libcudacxx/include/cuda/std/__concepts/concept_macros.h

```diff
@@ -109,11 +82,29 @@ _LIBCUDACXX_END_NAMESPACE_STD
 // _CUDA_VSTD macro is fully qualified.
 namespace __unqualified_cuda_std = _CUDA_VSTD; // NOLINT(misc-unused-alias-decls)
 
-#  if _CCCL_CUDACC_BELOW(12, 2)
-#    define _CCCL_CONCEPT_VSTD __unqualified_cuda_std // must not be fully qualified
-#  else
-#    define _CCCL_CONCEPT_VSTD _CUDA_VSTD
-#  endif
+#if _CCCL_CUDACC_BELOW(12, 2)
+#  define _CCCL_CONCEPT_VSTD __unqualified_cuda_std // must not be fully qualified
+#else
+#  define _CCCL_CONCEPT_VSTD _CUDA_VSTD
+#endif
+
+// clang-format off
```

Comment:
That was formatted fine before, what happened?

Reply:
it wasn't really fine before if you look. the diffs include:
```diff
-#  define _CCCL_TRAILING_REQUIRES(...)      ->__cccl_requires_t < __VA_ARGS__ _CCCL_TRAILING_REQUIRES_AUX_
+#  define _CCCL_TRAILING_REQUIRES(...)      -> __cccl_requires_t<__VA_ARGS__ _CCCL_TRAILING_REQUIRES_AUX_
```
the formatting before is a minor annoyance for me.